src/plugins/vis_types/xy/public/utils/get_series_params.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+import { ValueAxis, SeriesParam, ChartMode, InterpolationMode } from '../types';
+import { ChartType } from '../../common';
+import type { AggConfigs } from '../../../../data/public';
+
+const makeSerie = (
+  id: string,
+  label: string,
+  defaultValueAxis: ValueAxis['id'],
+  lastSerie?: SeriesParam
+): SeriesParam => {
+  const data = { id, label };
+  const defaultSerie = {
+    show: true,
+    mode: ChartMode.Normal,
+    type: ChartType.Line,
+    drawLinesBetweenPoints: true,
+    showCircles: true,
+    circlesRadius: 3,
+    interpolate: InterpolationMode.Linear,
+    lineWidth: 2,
+    valueAxis: defaultValueAxis,
+    data,
+  };
+  return lastSerie ? { ...lastSerie, data } : defaultSerie;
```

Comment:
nitpick: just wondering whether `defaultSerie` can be moved outside (without `data`) and this be simplified into:
```
return {...defaultSerie, ...lastSerie, data};
```

Reply:
Sure, done https://github.com/elastic/kibana/pull/112972/commits/74ce448df893e63738462e4d38d9aa87a70fc3a6